.gitmodules

```diff
@@ -1,18 +1,24 @@
 [submodule "renew-lib"]
 	path = renew-lib
-	url = https://github.com/renew-js/renew-lib.git
+	url = git@github.com:renew-js/renew-lib.git
```

Comment:
Why not https ?

Reply:
Because I'd like to use public key auth.